contribs/gmf/src/directives/partials/print.html

```diff
@@ -0,0 +1,69 @@
+      <p ng-if="customField.type === 'checkbox'"><input ng-model="customField.value" type="checkbox"> <span translate>Debug</span></p>
+      <input ng-if="customField.type === 'number'" ng-model="customField.value" class="form-control" type="number" placeholder="{{customField.name|translate}}">
+    </div>
+    <div ng-show="ctrl.fields.legend !== undefined" class="checkbox">
+      <label>
+        <input ng-model="ctrl.fields.legend" type="checkbox">
+        <span translate>Legend</span>
+      </label>
+    </div>
+    <div class="form-horizontal">
+      <div ng-show="ctrl.fields.layouts.length !== 1" class="form-group form-group-sm">
+        <label class="control-label col-md-3" translate>Layout</label>
+        <div class="col-md-9">
+            <button type="button" class="btn btn-default form-control" data-toggle="dropdown">{{ctrl.fields.layout | translate}}</button>
+          <ul class="dropdown-menu" role="menu">
+              <li ng-repeat="layout in ctrl.fields.layouts"><a href="" ng-click="ctrl.setLayout(layout)">{{layout | translate}}</a></li>
+          </ul>
+        </div>
+      </div>
+      <div ng-show="ctrl.fields.scales.length !== 1" class="form-group form-group-sm">
+        <label class="control-label col-md-3"translate>Scale</label>
+        <div class="col-md-9">
+            <button type="button" class="btn btn-default form-control" data-toggle="dropdown" aria-expanded="false">{{ctrl.fields.scale | translate}}</button>
+          <ul class="dropdown-menu" role="menu">
+            <li ng-repeat="scale in ctrl.fields.scales"><a href="" ng-click="ctrl.setScale(scale)">{{scale | translate}}</a></li>
+          </ul>
+        </div>
+      </div>
+      <div ng-show="ctrl.fields.dpis.length !== 1" class="form-group form-group-sm">
+        <label class="control-label col-md-3" translate>DPI</label>
+        <div class="col-md-5">
+          <button type="button" class="btn btn-default form-control" data-toggle="dropdown" aria-expanded="false">{{ctrl.fields.dpi}}</button>
+          <ul class="dropdown-menu" role="menu">
+            <li ng-repeat="dpi in ctrl.fields.dpis"><a href="" ng-click="ctrl.setDpi(dpi)">{{dpi}}</a></li>
+          </ul>
+        </div>
+      </div>
+      <div class="form-group form-group-sm">
+        <label class="control-label col-md-3" translate>Rotation</label>
+        <div class="col-md-5">
+          <input class="form-control" type="range" ng-model="ctrl.getSetRotation" ng-model-options="{getterSetter: true}" min="-180" max="180" style="border:0; background-color: transparent; box-shadow: none;" data-toggle="tooltip" title="{{'You can also use Alt+Shift on the map' | translate}}">
+        </div>
+        <div class="col-md-4">
+          <input class="form-control" type="number" ng-model="ctrl.getSetRotation" ng-model-options="{getterSetter: true}">
```

Comment:
The number may not be correctly displayed.
![screenshot from 2016-04-27 13 20 11](https://cloud.githubusercontent.com/assets/319774/14850689/0e17e60c-0c7b-11e6-8421-a1f1b26c0ad9.png)